enhancements/machine-api/aws-placement-groups.md

```diff
@@ -0,0 +1,472 @@
+---
+title: aws-placement-groups
+authors:
+  - "@JoelSpeed"
+reviewers:
+  - "@sdodson"
+  - "@deads2k"
+approvers:
+  - TBD
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - "@deads2k"
+creation-date: 2022-01-05
+last-updated: 2022-01-05
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - TBD
+---
+
+# AWS Placement Groups
+
+## Summary
+
+This enhancement describes the process of integrating [AWS Placement Groups](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/placement-groups.html#placement-groups-partition)
+into the OpenShift Machine API.
+
+Placement Groups allow users to define how groups of Machines should be placed within the Availability Zone.
+For example, they can clustered together within the same network spine, or separated for redundancy.
+
+## Motivation
+
+By adding support for Placement Groups within Machine API, end users will have finer control over the topology of their
+machines and will be able to leverage the benefits that different placement group strategies bring.
+
+### Goals
+
+- Allow users of Machine API to create a MachineSet with either a Cluster, Partition or Spread Placement Group
+- Allow usage of existing Placement Groups across 1 or more MachineSet
+- Create Placement Groups based on user configuration when they do not exist
+- Remove Placement Groups when they are no longer required
+- Ensure Placement Groups can be removed during the cluster deprovisioning process
+- Allow appropriate usage of Partition Placement Groups across multiple MachineSets.
+
+### Non-Goals
+
+- Allowing this feature to be leveraged during installation (day 0)
+  - Users will be able to modify MachineSets after manifest creation if required but this will not be exposed as a
+    part of the installation configuration before manifest generation
+- Support for Placement Groups for Control Plane Machines
+  - Users may replace their control plane at a later date using future automation if they wish to have their control
+    plane within a placement group
+- Support for moving Machines between placement groups
+
+## Proposal
+
+We will introduce new fields within the `Placement` section of the AWS Machine Provider Config which will allow users
+to specify which placement group to create instances within.
+
+If the placement group does not exist, it will be created based upon the group type and, if applicable partition count provided.
+
+We will also allow end users to specify which partition their instance should be created within when appropriate.
+
+### User Stories
+
+- As an OpenShift cluster administrator, I would like to use a spread placement group to ensure that my critical
+  workloads are unlikely to suffer from hardware failures simultanesouly.
+- As an operator of HPC workloads on OpenShift, I would like to use a cluster placement group to ensure that there is
```

Comment:
Yeah that's my bad, i should have spelled that out on first usage, will update. You're correct

Reply:
All good. Mostly I am wondering if we refer to this in prod docs with some other term, since it seems like the idea of HPC ought to have come up. But that's for me to dig in on :)